docs/components/board/_index.md

```diff
@@ -448,10 +448,11 @@ Get an [`DigitalInterrupt`](#digital_interrupts) by `name.`
 For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.digital_interrupt_by_name).
 
 ```python
-my_board = Board.from_robot(robot=robot, name=)
+my_board = Board.from_robot(robot=robot, name="my_board")
```

Comment:
Nice catch! No way `flake8` caught this one???

Reply:
it did tell me there was an error - I did manually have to fix it though